Tests/UnitTests/Core/Propagator/StepperTests.cpp

```diff
@@ -424,15 +425,16 @@ BOOST_AUTO_TEST_CASE(eigen_stepper_test) {
   freeParams.segment<3>(eFreePos0) *= 2;
   freeParams[eFreeTime] *= 2;
   freeParams.segment<3>(eFreeDir0) *= 2;
+  (freeParams.segment<3>(eFreeDir0)).normalize();
```

Comment:
This keeps the direction unchanged

Reply:
This test was stupid anyway, it was testing that if I created a non-well-defined freeParams with a non-normalised direction the `State` would do the normalization. I think we should remove this test and do garbage-in-garbage-out.